Configuration/DataProcessing/python/Impl/cosmicsEra_Run2_2018_hybrid.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+"""
+_cosmicsEra_Run2_2018_hybrid_
+
+Scenario supporting cosmic data taking
+
+"""
+
+import os
+import sys
+
+from Configuration.Eras.Era_Run2_2018_cff import Run2_2018
+from Configuration.DataProcessing.Impl.cosmics import cosmics
+
+class cosmicsEra_Run2_2018_hybrid(cosmics):
```

Comment:
the scenario names should have only an existing era name after ```Era_```.
The ```_hybridPrefix``` does not belong to an actual era.
Similar to ```trackingOnly``` and a few others, I think that there should be a ```cosmicsHybrid``` scenario not attached to any era so that the same scenario can be easily applied to the future years; alternatively, perhaps all changes can be defined in an Era so that this scenario becomes a trivial extension of a cosmics scenario like in Configuration/DataProcessing/python/Impl/cosmicsEra_Run2_2018.py